src/ar_http_iface_middleware.erl

```diff
@@ -211,19 +205,11 @@ handle(<<"POST">>, [<<"arql">>], Req, Pid) ->
 		{ok, QueryJson, ReadReq} ->
 			case ar_serialize:json_struct_to_query(QueryJson) of
 				{ok, Query} ->
-					TXs = ar_util:unique(ar_parser:eval(Query)),
-					case TXs of
-						[] -> {200, #{}, [], ReadReq};
-						Set ->
-							{
-								200,
-								#{},
-								ar_serialize:jsonify(
-									ar_serialize:hash_list_to_json_struct(Set)
-								),
-								ReadReq
-							}
-					end;
+					TXIDs = ar_util:unique(ar_parser:eval(Query)),
+					TXBlockHeightPairs = lists:map(fun ar_tx_search:get_tx_block_height/1, TXIDs),
+					SortedTXs = [TXID || {_, TXID} <- lists:sort(TXBlockHeightPairs)],
```

Comment:
I wonder if it makes more sense from the UX perspective to sort in the descending order so that the newest items are displayed on top, like we do in `GET /wallet/[Addr]/txs`, what do you think?

Reply:
Makes sense. We should definitely be consistent and I think we'll tend to search for recent data so I'll update the function